lib/templates/bibliotheek/boek.tpl

```diff
@@ -94,151 +75,146 @@
+						{if !$boek->isEigenaar($exemplaar->id) AND ($exemplaar->eigenaar_uid!='x222' OR $total_exemplaren_bibliotheek>0 )}
+							{CsrDelft\model\ProfielModel::getLink($exemplaar->eigenaar_uid, 'pasfoto')}
+							<div class="statusmarkering"><span class="biebindicator {$exemplaar->status}" title="Boek is {$exemplaar->status}">• </span></div>
 						{/if}
-						{if $exemplaar.eigenaar_uid=='x222'}
+						{if $exemplaar->eigenaar_uid=='x222'}
 							{assign var=total_exemplaren_bibliotheek value=$total_exemplaren_bibliotheek+1}
 						{/if}
 					{/foreach}
 					<br /><div class="clear"></div>
-					<label>&nbsp;</label><a onclick="jQuery(this).parent().parent().children('div.exemplaar.uitgebreid').show(); jQuery(this).parent().remove();">» meer informatie</a>
+					<label>&nbsp;</label><a onclick="jQuery(this).parent().parent().children('div.exemplaar->uitgebreid').show(); jQuery(this).parent().remove();">» meer informatie</a>
 				</div>
 			{/if}
 		</div>
-	</div>
-	{if $boek->isEigenaar()}
-		{* javascript invoegen van de fields *}
-		{$boek->ajaxformuliervelden->getScriptTag()}
-	{/if}
 
 	{* beschrijvingen *}
 
-	<div class="beschrijvingen">
+	<div class="beschrijvingen col-md-6">
 		<h3 class="header">Recensies en beschrijvingen</h3>
-		{if $boek->countBeschrijvingen()>0}
 			<table id="beschrijvingentabel">
-			{foreach from=$boek->getBeschrijvingen() item=beschr}
-				{assign var=beschrijving value=$beschr->getBeschrijving()}
+			{foreach from=$recensies item=beschrijving}
+
 				<tr><td class="linkerkolom"></td><td style="width:506px"></td></tr>
 				<tr >
-					{if isset($beschrijving.bewerk)}
-						<td colspan="2">
-							{* formulier voor toevoegen/bewerken van beschrijvingen *}
-							{$boek->getFormulier()->view()}
-						</td>
-					{else}
 						<td class="linkerkolom recensist">
-							<span class="recensist">{CsrDelft\model\ProfielModel::getLink($beschrijving.schrijver_uid, 'civitas')}</span><br />
-							<span class="moment">{$beschrijving.toegevoegd|reldate}</span><br />
+							<span class="recensist">{CsrDelft\model\ProfielModel::getLink($beschrijving->schrijver_uid, 'civitas')}</span><br />
+							<span class="moment">{$beschrijving->toegevoegd|reldate}</span><br />
 
 						{* knopjes bij elke post *}
-							{if $boek->magBeschrijvingVerwijderen($beschrijving.id)}
-								{knop url="/bibliotheek/bewerkbeschrijving/`$boek->getId()`/`$beschrijving.id`#Beschrijvingsformulier" type=bewerken}
-								{knop url="/bibliotheek/verwijderbeschrijving/`$boek->getId()`/`$beschrijving.id`" type=verwijderen confirm='Weet u zeker dat u deze beschrijving wilt verwijderen?'}
+							{if $beschrijving->magVerwijderen()}
+								{knop url="/bibliotheek/verwijderbeschrijving/`$boek->getId()`/{CsrDelft\model\security\LoginModel::getUid()}" class='post ReloadPage' type=verwijderen confirm='Weet u zeker dat u deze beschrijving wilt verwijderen?'}
```

Comment:
Gekke syntax, moet het niet `{boek->getId()}` zijn?

Reply:
Dat kan ook inderdaad. Ik heb dingen die nog werkten zoveel mogelijk gelaten zoals ze waren.

Reply:
Ik zie in de docs van smarty dat je backticks mag gebruiken, maar dat is voor als je al tussen {} zit, maar in deze regel wordt ook gewoon geneste {} gebruikt. 

Als het werkt lijkt het met goed, de smarty zal hopelijk toch niet lang blijven bestaan.